tests/dbsclient_t/validation/DBSValidation_t.py

```diff
@@ -351,7 +350,14 @@ def check(input, output):
                         print("--------output description--------")
                         print(output[key])
                     """
+                    if key not in output:
+                        print("no %s found in output" % key)
                     self.assertTrue(key in output)
+                    if not output[key]:
+                        print("no values for %s" % key)
+                        if value:
+                            print("key %s has valut %s in input data" % (key, value))
+#                             self.assertEqual(value, output[key])
```

Comment:
@yuyiguo originally I added assert here, but I found that in this case the existing tests will fail against testbed instance running Python server. The check which didn't pass was processing_era description key. Since the code never checked it I think it was a miss, therefore I removed assert to keep code fully compatible with existing server.